Engine/Communication/robot_state.py

```diff
@@ -1,5 +1,5 @@
 from collections import namedtuple
 
-RobotPacket = namedtuple('RobotPacket', 'robot_id command kick_type kick_force dribbler_active charge_kick')
+RobotPacket = namedtuple('RobotPacket', 'robot_id command kick_type kick_force dribbler_active dribbler_speed charge_kick')
```

Comment:
Dribbler_active condition is now useless with the stopped constant.

Reply:
dribbler_active -> Do we send a command to the dribbler
dribbler_speed -> Is command forcing the dribbler to spin or forcing to stop spinning?
The names are now a bit confusing

Reply:
We should remove `dribbler_active` and add a constant `DibblerState.Automatic` and so on.